JSDemos/Demos/Toolbar/Multiline/React/data.js

```diff
@@ -0,0 +1,102 @@
+export const colors = [
```

Comment:
It is necessary here?

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/c8c3b387ef97137bad2b5858c6be85cfff992081